tests/umock_c_reals_int/umock_c_reals_int.c

```diff
@@ -177,4 +177,17 @@ TEST_FUNCTION(no_rename_to_Real_if_ENABLE_MOCKS_is_not_defined)
     ASSERT_ARE_EQUAL(int, 42, result);
 }
 
+/* Tests_SRS_UMOCK_C_LIB_01_220: [ `UMOCK_REAL` shall produce the name of the real function generated by umock. ]*/
+TEST_FUNCTION(calling_a_real_function_is_possible)
+{
+    // arrange
+    int result;
+
+    // act
+    result = UMOCK_REAL(test_dependency_no_args());
```

Comment:
>UMOCK_REAL(test_dependency_no_args [](start = 13, length = 34)

you mean UMOCK_REAL(test_dependency_no_args)();?

The macro definition doesn't say anything about execution, it would just replace one name with another.

Reply:
Fixed, cheers!